app.js

```diff
@@ -1,4 +1,8 @@
-const { getTopics, getEndpoints } = require("./2_Controllers/test_controller");
+const {
+  getTopics,
+  getEndpoints,
+  getArticlesById,
+} = require("./2_Controllers/test_controller");
 const express = require("express");
 const app = express();
 
```

Comment:
There is no need to include `app.use(express.json());` until your API handles either a PATCH or POST (something with a request body)